### PR TITLE
Commented all documentation related with unregister feature

### PR DESCRIPTION
**Pull request by echavezNS**
As I saw in https://github.com/bower/bower/issues/2210 that the **feature** of `unregister` is disabled, so I just commented all the lines I found in the documentation related with **unregister** command.

### CODE REVIEW

**Comment by riyadhalnur** [comment]
Thank you for the PR. I would suggest removing the lines instead of commenting them out.

**Comment by echavezNS** [comment]
Perfect, I did a new commit, should I do a rebase?

File: docs/about.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ Bower is **Bower** with a capital B.
 
 <p><a href="/img/bower-logo.png"><img class="download-logo" src="/img/bower-logo.png" alt="Bower logo" /></a></p>
 
-Download [**PNG**](/img/bower-logo.png) · [**SVG**](/img/bower-logo.svg) · [**EPS**](/img/bower-logo.eps)
+Download [**PNG**](/img/bower-logo.s) · [**SVG**](/img/bower-logo.svg) · [**EPS**](/img/bower-logo.eps)
```
Comment (riyadhalnur) [inline docs/about.md]: typo in the filename here

**Comment by echavezNS** [comment]
I didn't see that, let me change it  :lollipop:

**Comment by riyadhalnur** [comment]
@sheerun and @bower/contributors Your thoughts?

**Comment by accommodavid** [comment]
Maybe change "`unregister` command is temporarily disabled" into "The `unregister` command is temporarily disabled" so it starts with a capital letter? Otherwise LGTM

**Comment by jamesxv7** [comment]
Because at the moment is only a temporary action, as per #2210, completely remove the section seems somewhat abrupt. I understand it is more convenient to place a note stating that by the time the option is disabled and refers to the Github issue for details.

**Comment by echavezNS** [comment]
So, should I delete the section and let it like this?

"You’ll likely want to bower cache clean after your change. You can request a package to be unregistered manually."

**Comment by echavezNS** [comment]
How about this @jamesxv7 @riyadhalnur  @accommodavid ?

File: docs/creating-packages.md [hunk 2]
```diff
@@ -64,17 +64,4 @@ Please do not squat on package names. Register your package and claim your name
...
-bower login
-# enter username and password
-? Username:
-? Password:
-# unregister packages after successful login
-bower unregister <package>
-{% endhighlight %}
-
-You'll likely want to [`bower cache clean`](/docs/api#cache-clean) after your change. Please remember it is generally considered bad behavior to remove versions of a library that others are depending on. Think twice :) If the above doesn't work for you, you can [request a package be unregistered manually](https://github.com/bower/bower/issues/120).
+You'll likely want to [`bower cache clean`](/docs/api#cache-clean) after your change. By the moment the command `unregister` is disabled, check [#2210](https://github.com/bower/bower/issues/2210) issue for more information. However, you can [request a package to be unregistered manually](https://github.com/bower/bower/issues/120).
```
Comment (riyadhalnur) [inline docs/creating-packages.md]: Should be 

> At the moment, the `unregister` command is temporarily disabled. You can check out this issue - #2210, for more information.

**Comment by riyadhalnur** [comment]
Just a small language in the language. Everything else looks good.

**Comment by echavezNS** [comment]
Done, I hope everything looks good. @riyadhalnur

**Pull request merged**